opmd_viewer/openpmd_timeseries/plotter.py

```diff
@@ -195,6 +204,10 @@ def show_field_2d(self, F, info, slicing_dir, m,
 
         geometry: string
            Either "2dcartesian", "3dcartesian" or "thetaMode"
+
+        plot_range : list of lists
```

Comment:
Yeah, I agree there could be different options here. I was following the same convention as [matplotlib's hist2d](https://matplotlib.org/devdocs/api/_as_gen/matplotlib.axes.Axes.hist2d.html).

Since this convention is also used by the other upcoming pull requests, would you be okay with keeping your remark in mind, and then maybe dealing with it, in bulk, once the 3 upcoming pull requests are merged?

If you are